### PR TITLE
Minor refactor of 2 bin scripts since they where bugging me

### PR DESCRIPTION
**Pull request by iljo-dp**

### CODE REVIEW

File: bin/omakub-font [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
-	NERD_FONT="JetBrainsMono NFM"
-elif [ "$FONT" == "meslo" ]; then
-	NERD_FONT="MesloLGLDZ Nerd Font"
-fi
+declare -A NERD_FONTS=(
+  ["cascadia-mono"]="CaskaydiaMono Nerd Font"
+  ["fira-mono"]="FiraMono Nerd Font"
+  ["jetbrains-mono"]="JetBrainsMono NFM"
+  ["meslo"]="MesloLGLDZ Nerd Font"
+)
```
Comment (jeremy) [inline bin/omakub-font:10]: Could see using `case` instead. Prefer syntax over a data type for this usage.

File: bin/omakub-font [hunk 2]
```diff
@@ -1,18 +1,15 @@
-# Pick a preconfigured theme
```
Comment (jeremy) [inline bin/omakub-font:1]: Don't remove comments.

**Review by jeremy — CHANGES_REQUESTED** [review]

**Comment by iljo-dp** [comment]
your requested changes are now present

File: bin/omakub-font (hunk 1 above)
Comment (iljo-dp) [reply bin/omakub-font]: fixed

File: bin/omakub-theme [hunk 3]
```diff
@@ -1,39 +1,29 @@
...
-[ ! -n "$THEME" ] && exit 0
+[ -z "$THEME" ] && exit 0
 
 # Install theme in Gnome, Terminal, and both default editors
-source $OMAKUB_PATH/themes/gnome/$THEME.sh
-cp $OMAKUB_PATH/themes/alacritty/$THEME.toml ~/.config/alacritty/theme.toml
-cp $OMAKUB_PATH/themes/neovim/$THEME.lua ~/.config/nvim/lua/plugins/theme.lua
+source "$OMAKUB_PATH/themes/gnome/$THEME.sh"
+cp "$OMAKUB_PATH/themes/alacritty/$THEME.toml" ~/.config/alacritty/theme.toml
+cp "$OMAKUB_PATH/themes/neovim/$THEME.lua" ~/.config/nvim/lua/plugins/theme.lua
```
Comment (dhh) [inline bin/omakub-theme:11]: don't see why we need this quoting? The value of $THEME is known to be safe.

Comment (iljo-dp) [reply bin/omakub-theme]: yep my bad, it's a bad habit I got

File: bin/omakub-theme [hunk 4]
```diff
@@ -1,8 +1,9 @@
 # Pick a preconfigured theme
 THEME_NAMES=("Tokyo Night" "Catppuccin" "Nord" "Everforest" "Gruvbox" "Kanagawa" "Rose Pine")
+
```
Comment (dhh) [inline bin/omakub-theme:3]: Not needed

File: bin/omakub-font [hunk 5]
```diff
@@ -1,18 +1,16 @@
...
+  "cascadia-mono") NERD_FONT="CaskaydiaMono Nerd Font" ;;
+  "fira-mono") NERD_FONT="FiraMono Nerd Font" ;;
+  "jetbrains-mono") NERD_FONT="JetBrainsMono NFM" ;;
+  "meslo") NERD_FONT="MesloLGLDZ Nerd Font" ;;
+  *) exit 1 ;;
+esac
 
 gsettings set org.gnome.desktop.interface monospace-font-name "$NERD_FONT 10"
-cp $OMAKUB_PATH/fonts/alacritty/$FONT.toml ~/.config/alacritty/font.toml
+cp "$OMAKUB_PATH/fonts/alacritty/$FONT.toml" ~/.config/alacritty/font.toml
```
Comment (dhh) [inline bin/omakub-font:15]: Not needed

File: bin/omakub-theme [hunk 6]
```diff
@@ -11,29 +12,18 @@ cp $OMAKUB_PATH/themes/neovim/$THEME.lua ~/.config/nvim/lua/plugins/theme.lua
...
+  "tokyo-night") VSC_THEME="Tokyo Night enkia.tokyo-night" ;;
+  "everforest") VSC_THEME="Everforest Dark sainnhe.everforest" ;;
+  "rose-pine") VSC_THEME="Rosé Pine Dawn mvllow.rose-pine" ;;
+  "nord") VSC_THEME="Nord arcticicestudio.nord-visual-studio-code" ;;
+  "kanagawa") VSC_THEME="Kanagawa qufiwefefwoyn.kanagawa" ;;
+  *) exit 1 ;;
+esac
+
+code --install-extension "${VSC_THEME##* }" > /dev/null
+sed -i "s/\"workbench.colorTheme\": \".*\"/\"workbench.colorTheme\": \"${VSC_THEME% *}\"/g" ~/.config/Code/User/settings.json
```
Comment (dhh) [inline bin/omakub-theme:27]: I find using a single string too clever here. Makes it harder to read.

**Comment by iljo-dp** [comment]
I've tried to at least fix the problem by simplifying the sed command, but I don't fully understand the problem you are facing

**Pull request closed without merging**